src/System.ComponentModel.Composition/src/Microsoft/Internal/Runtime/Serialization/SerializationServices.cs

```diff
@@ -1,21 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Runtime.Serialization;
-
-namespace Microsoft.Internal.Runtime.Serialization
-{
-#if FEATURE_SERIALIZATION
```

Comment:
you don't need to bring back this helper class as it's meaningless. By calling `base.GetObjectData` in the types `GetObjectData` implementation at the beginning like in other exceptions you already check if the `SerializationInfo` is null. I can't come up with a meaning for the null check of name as we provide it ourselves in the types implementation. Therefore I suggest not bringing this class back.